src/main.cpp

```diff
@@ -134,6 +137,7 @@ static void lPrintVersion() {
     printf("    [--host-stub <filename>]\t\tEmit host-side offload stub functions to file\n");
     printf("    [-h <name>/--header-outfile=<name>]\tOutput filename for header\n");
     printf("    [-I <path>]\t\t\t\tAdd <path> to #include file search path\n");
+    printf("    [--ignorecpp]\t\t\tSuppress errors from the preprocessor (if enabled)\n");
```

Comment:
Other names welcome.

Reply:
Let's go with `--ignore-preprocessor-errors`.
My reasoning is the following: `cpp` stands for "C PreProcessor" only for very limited number of people. For the rest it's simply confusing. So any other naming will be confusing and not clear without help message.
And `-ignorecpp` is especially confusing. Could you please change internal var name to `ignoreCPPErrors` as well?